lib/hammer_cli_foreman/host.rb

```diff
@@ -66,6 +67,8 @@ def extend_data(host)
         field nil, _("Puppet Master Proxy"), Fields::SingleReference, :key => :puppet_proxy
         field :certname, _("Cert name")
         field :managed, _("Managed"), Fields::Boolean
+        field :build_status_label, _("Build Status")
```

Comment:
Would it make sense to encapsulate both statuses with `label _('Status') do`?
It would probably make it more readable to have all statuses in one section. Also it would be easier for plugins to add their own statuses. That should be soon enabled by https://github.com/theforeman/hammer-cli/pull/275.